arch/arch-install

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/bash
+
+# This is my custom installation script of Arch Linux. I will recommend reading
+# the commands that Im using before using it, because I hard coded some values
+# that you might not need/use. TODO tags are mostly things that you might need
+# to change.
+
+set -euo pipefail
+
+# --------------------------------- Functions ----------------------------------
+
+Err()
+{
+    printf 'Err: %s\n' "$2" 1>&2
+    # shellcheck disable=2086
+    (($1 > 0)) && exit $1
+}
+
+# https://stackoverflow.com/a/3352015
+
+TrimStart()
+{
+    echo "${1#"${1%%[![:space:]]*}"}"
```

Comment:
Iwctl output always has a `^[[0m` at the start of every line, which stop `TrimStart` from removing spaces.

Reply:
There is (most likely) 3 solutions for this:

1. The easiest, just offset the output lines by 4 ( ${Line:4} ).
2. Change the delimiter of `readarray` to `^[[0m`.
3. `TrimStart` ( and `TrimEnd` ) should remove any non-alphanumeric character, instead of only spaces.